test-common-blackduck/src/main/java/com/synopsys/integration/alert/test/common/processor/api/BlackDuckResponseTestUtility.java

```diff
@@ -0,0 +1,97 @@
+package com.synopsys.integration.alert.test.common.processor.api;
```

Comment:
This package should probably be `com.synopsys.integration.alert.test.common.blackduck` now.